### PR TITLE
Remove SQLAlchemy models

### PR DESCRIPTION
**Pull request by FedericoCeratto**
Remove SQLalchemy models from list_files
Add integration tests
Speed improvements: count(*) should be faster now
Bump up requirements

### CODE REVIEW

**Comment by FedericoCeratto** [comment]
(still WIP)

File: measurements/api/measurements.py [hunk 1]
```diff
@@ -206,41 +221,43 @@ def get_measurement(measurement_id, download=None):
...
-    except NoResultFound:
-        # XXX we should actually return a 404 here
-        raise BadRequest("No measurement found")
+    )
+    where = sql.text("measurement.msm_no = :msm_no")
+    query = select(cols).where(where).select_from(table)
+    query_params = dict(msm_no=msm_no)
+    q = current_app.db_session.execute(query, query_params)
+
+    msmt = q.fetchone()
```
Comment (hellais) [inline measurements/api/measurements.py]: Should be noted we aren't handling the case in which there are duplicate results. We may or may not care about that.

In theory it's easy to fix by just doing an additional call to `fetchone()` and ensuring we get `None` on the second call.

File: measurements/api/private.py [hunk 2]
```diff
@@ -165,169 +161,42 @@ def api_private_countries():
...
-                "probe_asn": "AS{}".format(row.probe_asn),
-                "test_start_time": row.test_start_time,
-                "input": row.input,
-            }
-        )
 
-    return jsonify({"results": results})
+@api_private_blueprint.route("/blockpages", methods=["GET"])
+def api_private_blockpages():
+    abort(404)
```
Comment (hellais) [inline measurements/api/private.py]: Maybe we also want to log this in a structured way? Though OTOH perhaps 404 is enough as we can easily parse nginx logs looking for that.

File: measurements/api/measurements.py (hunk 1 above)
Comment (FedericoCeratto) [reply measurements/api/measurements.py]: Until we introduce OOID we cannot pinpoint msmts and get_measurement is simply returning the row.

File: measurements/api/private.py (hunk 2 above)
Comment (FedericoCeratto) [reply measurements/api/private.py]: We are not logging other calls e.g. /test_names (except from the Nginx side). Also the deprecated calls seemed to have no traffic at all.

**Comment by FedericoCeratto** [comment]
@hellais updated after removing most of models.py and rerunning tests

**Pull request merged**